src/org/omegat/languagetools/LanguageToolNetworkBridge.java

```diff
@@ -227,29 +222,18 @@ public void destroy() {
         }
 
         List<Map<String,Object>> matches = (List) response.get("matches");
-        ArrayList<LanguageToolMatch> ltMatches = new ArrayList<>();
+        List<Mark> r = new ArrayList<>();
 
         matches.stream().forEach((match) -> {
```

Comment:
Rather than create a new `ArrayList` and repeatedly add to it in a `forEach` I would recommend mapping and collecting directly to a list, like:

```
return matches.stream().map(match -> /* create and return new Mark */).collect(Collections.toList())
```